src/tests/example.spec.js

```diff
@@ -24,4 +22,69 @@ test.describe('', () => {
         await shopingCartPage.removeCartItemById(0);
         await expect(shopingCartPage.cartItems).not.toBeAttached();
     });
+
+    test('Should have correct basic sorting list on the Inventory page', async ({ inventoryPage}) => {
+        await test.step('Sorting products list', async () => {
+            await inventoryPage.sortingBtn.click();
+
+            const sortList = inventoryPage.sortingBtn.locator('option');
+            await expect(sortList).toHaveText([
+                'Name (A to Z)',
+                'Name (Z to A)',
+                'Price (low to high)',
+                'Price (high to low)']);
+        });
+    });
+
+
+    const sortingOptions = [`az`, 'za', 'lohi', 'hilo']
+
+    for (const option of sortingOptions) {
+        test(`Should correct sorting with type: ${option}`, async ({inventoryPage}) => {
+        await inventoryPage.sortingBtn.click();
+        await inventoryPage.switchSorting(option);
+
+        const list = await inventoryPage.itemsPrice;
+        const priceList = await inventoryPage.chosenProducts(list);
+
+        priceList.forEach(async (data, index) => {
+            await expect(list.nth(index)).toHaveText(data);
```

Comment:
Sorry I didn't get what this test verifies. Let's have a short call to discuss it?